e2e/fixtures/ethereum.ts

```
@@ -0,0 +1,183 @@
+const account = privateKeyToAccount(
+    "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
+);
+
+type EthereumFixtures = {
+    walletClient: ReturnType<typeof createWalletClient>;
+    injectProvider: () => Promise<void>;
+};
+
+export const test = base.extend<EthereumFixtures>({
+    // eslint-disable-next-line no-empty-pattern
+    walletClient: async ({}, use) => {
```

Biome (2.3.13)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)